embassy-stm32/src/bdma/v2.rs

```diff
@@ -0,0 +1 @@
+pub(crate) unsafe fn init() {}
```

Comment:
I think BDMAv1 and BDMAv2 should use the same code, as they are exactly identical except CSELR. The CSELR bits can do `#[cfg(bdma_v2)]`

Same with DMAv1/DMAv2

Reply:
Agreed